packages/langium/src/test/langium-test.ts

```diff
@@ -55,6 +52,22 @@ export function expectSymbols(services: LangiumServices, cb: ExpectFunction): (i
     };
 }
 
+export function expectFoldings(services: LangiumServices, cb: ExpectFunction): (input: ExpectedBase) => void {
```

Comment:
General remark: Wouldn't it be nicer to use a more expressive local name for the callback, like `expectEqual`?